docs/new-audits.md

```diff
@@ -31,8 +31,26 @@ An audit can return a number of different [detail types](https://github.com/Goog
 | `'url'`                   | Network Resource      | we will make it a pretty link          |
 | `'thumbnail'`             | Image Resource        | same as above, but we show a thumbnail |
 | `'link'`                  | -                     | arbitrary link / url combination       |
+| `'bytes'`                 | -                     | value is in bytes but formatted as KiB |
 | `'text'\|'ms'\|'numeric'` | -                     |                                        |
 
+### Granularity
+
+The following detail types accept a `granularity` field:
+
+- `bytes`
+- `ms`
+- `numeric`
+
+`granularity` must either be a positive integer or power of 10. Some examples of valid values for `granularity`:
```

Comment:
When you say "power of 10" you mean "integer power of 10" right? If so I don't think 0.5 qualifies.

Reply:
updated but... maybe we could just reduce the restriction on granularity to any positive number?